mscr-ui/src/pages/mscr-style-customizations.scss

```diff
@@ -28,6 +28,9 @@ body {
   }
 }
 
+.fi-select-item-list_content_wrapper:has(li[id$="SHACL"]) {
+  max-height: 160px !important;
```

Comment:
Should the "important" be dropped now that you added specificity? I tested and this new rule seems to override even without "important".

Reply:
Good observation. Unnecessary !important style definition removed.